docs/UserGuide.md

```diff
@@ -2,8 +2,41 @@
 > - This guide assumes that the reader is familiar with using [Java](https://www.java.com/)
 
 # Table of Contents
-
-{:toc}
+<!-- TOC -->
```

Comment:
Thanks for this! Do you have any ideas why `{:toc}` doesn't work for UG? 😅

Reply:
Im not sure, I used the Intellij `insert` short cut. Maybe github's md has some issues HAHA